app/internal/on_this_day_events.py

```diff
@@ -0,0 +1,16 @@
+import json
+from datetime import datetime
+
+import requests
+
+
+def get_on_this_day_events():
+    now = datetime.now()
+    day = now.day
+    month = now.month
+
+    res = requests.get(
```

Comment:
Prefer prefetching this one time only.

Populate the database with the relevant info using some script.
Running `requests.get` on our server is a blocking call, and will probably not perform well on multiple-users environment.